src/kinetics/ElectronCollisionPlasmaRate.cpp

```diff
@@ -140,6 +176,24 @@ void ElectronCollisionPlasmaRate::setContext(const Reaction& rxn, const Kinetics
         throw InputFileError("ElectronCollisionPlasmaRate::setContext", rxn.input,
             "ElectronCollisionPlasmaRate requires one and only one electron");
     }
+
+    if (!rxn.reversible) {
+        return; // end checking of forward reaction
+    }
+
+    // For super-elastic collisions
+    if (rxn.products.size() != 2) {
+        throw InputFileError("ElectronCollisionPlasmaRate::setContext", rxn.input,
+            "ElectronCollisionPlasmaRate requires exactly two products"
+            "if the reaction is reversible (super-elastic collisions)");
+    }
+
+    // Must have only one electron
+    if (rxn.products.at(electronName) != 1) {
+        throw InputFileError("ElectronCollisionPlasmaRate::setContext", rxn.input,
+            "ElectronCollisionPlasmaRate requires one and only one electron in products"
+            "if the reaction is reversible (super-elastic collisions)");
+    }
```

Comment:
These need a space inside the string so it doesn't say "productsif" when printed:
```suggestion
            "ElectronCollisionPlasmaRate requires exactly two products"
            " if the reaction is reversible (super-elastic collisions)");
    }

    // Must have only one electron
    if (rxn.products.at(electronName) != 1) {
        throw InputFileError("ElectronCollisionPlasmaRate::setContext", rxn.input,
            "ElectronCollisionPlasmaRate requires one and only one electron in products"
            " if the reaction is reversible (super-elastic collisions)");
    }
```